CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+
+- Added two new meeting events: `sendingAudioFailed` to indicate a failure in sending audio packets out and `sendingAudioRecovered` to indicate recovery after a `sendingAudioFailed` event
```

Comment:
Not sure if this was discussed, did we consider `audioSendXXX` or `audioSendingXXX`? Feels more natural to namespace the events that way.

...and now i see receivingAudioDropped...but also audioInputFailed....argh. I'm still inclined to ignore `receivingAudioDropped` and prefix with `audio` first, what do you think?

Reply:
Yes this was discussed and the names were suggested by @simmkyu. I personally feel `sendingAudioFailed` should tie up closely with `receivingAudioDropped` because it's related to transmission/reception as opposed to something like `audioInputXXX`. That said, this is not a strong opinion of mine so if any other reviewer agrees I will change it to `audioSendingXXX`